torch/_inductor/fx_passes/post_grad.py

```diff
@@ -114,6 +116,41 @@ def mm_plus_mm(match: Match, mat1, mat2, mat3, mat4):
     return inductor.kernel.mm_plus_mm.tuned_mm_plus_mm(mat1, mat2, mat3, mat4)
 
 
+@register_graph_pattern(
+    CallFunction(
+        aten.cumsum.default,
+        CallFunction(
+            prims.convert_element_type.default,
```

Comment:
presumably this is amp - could register a non amp one as well

Reply:
I don't think it is amp.  The conversion is actually float32->int64.  No idea why it is there....